test/retainFn-spec.ts

```diff
@@ -1,4 +1,4 @@
-import retainFn from '../src/retainFn';
+import {retainFn} from '../src/retainFn';
```

Comment:
nit spacing in braces for style

Reply:
Hmm this is actually the style I want, to make goog standards. 
I'll update the rule to disallow spacing in brackets.